.github/workflows/build_and_test.yml

```diff
@@ -201,6 +173,92 @@ jobs:
         name: unit-tests-log-${{ matrix.modules }}-${{ matrix.comment }}-${{ matrix.java }}-${{ matrix.hadoop }}-${{ matrix.hive }}
         path: "**/target/unit-tests.log"
 
+  pyspark:
+    name: "Build modules: ${{ matrix.modules }}"
+    runs-on: ubuntu-20.04
+    container:
+      image: dongjoon/apache-spark-github-action-image:20201015
```

Comment:
@dongjoon-hyun, quick question. what about we just set this image for the main job above (meaning use it for all tests and build), and remove this new job here? I think that'll be easier to move to use Docker image for all test cases.

Reply:
@HyukjinKwon . I tried already. Please see https://github.com/apache/spark/pull/30059#issuecomment-709555763 .

Reply:
That's the reason why I refocus on this only which is most time-saving one.
We can revisit the others later.

Reply:
👌